### PR TITLE
feat: beautify auth page after pkce login

### PR DESCRIPTION
**Pull request by jpinkney-aws**
## Problem
- Old screen was ugly and basic

## Solution
- New screen is similiar to the current oidc login webpage

## Open questions/TODOs:
- Should we have separate error and main pages?
- Missing this icon above the approval message (is it even needed?)
Screenshot 2024-04-18 at 5 48 29 PM

### Request approved
Screenshot 2024-04-18 at 5 32 37 PM

### Request denied with a reason
Screenshot 2024-04-18 at 5 33 48 PM

### Someone loaded the main page without specifying productName and scopes
Screenshot 2024-04-18 at 5 33 12 PM

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/core/src/auth/sso/media/index.html [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+            <div style="font-size: 3rem" class="center-icon aws-icon">
+                <svg
+                    id="Layer_1"
+                    data-name="Layer 1"
+                    xmlns="http://www.w3.org/2000/svg"
+                    width="6rem"
+                    :height="`${6 * 0.61}rem`"
+                    viewBox="0 0 50 30"
+                >
+                    <path
```
Comment (jpinkney-aws) [inline packages/core/src/auth/sso/media/index.html:21]: This is the AWS icon, attempting to make it an icon and then importing it makes it lose colour :(

File: packages/core/src/auth/sso/media/auth.css
Comment (justinmk3) [inline packages/core/src/auth/sso/media/auth.css:1]: can this live in `packages/*/resources` like all other (I think) css files? media/ is a new pattern and ideally we should be consistent where possible, that helps other developers follow things intuitively

File: packages/core/src/auth/sso/server.ts [hunk 2]
```diff
@@ -76,7 +82,22 @@ export class AuthSSOServer {
...
+                        await this.loadResource(res, iconPath)
+                        break
+                    }
+                    const resourcePath = path.join(
+                        globals.context.extensionUri.fsPath,
+                        'dist',
+                        'src',
+                        'auth',
+                        'sso',
+                        'media',
```
Comment (justinmk3) [inline packages/core/src/auth/sso/server.ts:96]:
```suggestion
                        'dist/src/auth/sso/media',
```

File: packages/toolkit/scripts/build/copyFiles.ts [hunk 3]
```diff
@@ -32,6 +32,10 @@ const tasks: CopyTask[] = [
     { target: path.join('../core', 'resources'), destination: path.join('..', 'resources') },
     { target: path.join('../core', 'package.nls.json'), destination: path.join('..', 'package.nls.json') },
     { target: path.join('../core', 'src', 'templates'), destination: path.join('src', 'templates') },
+    {
+        target: path.join('../core', 'src', 'auth', 'sso', 'vue'),
+        destination: path.join('src', 'auth', 'sso', 'vue'),
```
Comment (justinmk3) [inline packages/toolkit/scripts/build/copyFiles.ts:37]: need need to use join() on literals
```suggestion
        target: '../core/src/auth/sso/vue',
        destination: 'src/auth/sso/vue',
```

**Review by justinmk3 — APPROVED** [review]

**Comment by justinmk3** [comment]
CI failures look related

**Pull request merged**